web/src/main/webapp/WEB-INF/config-spring-geocat.xml

```diff
@@ -67,7 +67,7 @@
         <property name="descColumn" value="DESC"/>
         <property name="searchColumn" value="SEARCH"/>
         <property name="geoIdColumn" value="GEO_ID"/>
-        <property name="srs" value="EPSG:21781"/>
+        <property name="srs" value="EPSG:4326"/>
```

Comment:
This is how to configure geocat to handle world wide geometry (purpose of this PR).